packages/inline-choice/src/event-helper.js

```diff
@@ -0,0 +1,15 @@
+export function dispatchModelSetEvent(eventObj, data) {
```

Comment:
instead of this you can use: `@pie-libs/pie-player-events@^0.0.1` which contains: 
`import {ModelSetEvent, SessionChangedEvent} from '@pie-libs/pie-player-events`

Reply:
Done